### PR TITLE
165: focus button fix

### PR DESCRIPTION
**Pull request by JordanDDisch**

### CODE REVIEW

**Comment by rcaracaus** [comment]
Should work in Twig builder as well.

I don't really like how the fullscreen mode does not let you test screens that are bigger

Some tests are failing.

**Comment by rcaracaus** [comment]
Is there a way to ignore libraries with eslint?

**Comment by JohnAlbin** [comment]
> Is there a way to ignore libraries with eslint?

Yep. There's an `.eslintigore` file. But it already has `builder/*/kss-assets/*.js` in it.

**Comment by rcaracaus** [comment]
@JordanDDisch this should be in Twig builder too

**Comment by JordanDDisch** [comment]
@JohnAlbin Can you test this and see what else needs work? Works with the both handlebars and twig generators.

File: generator/handlebars/template/index.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<!--[if lte IE 6]>            <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en"> <![endif]-->
+<!--[if IEMobile 7]>          <html class="no-js iem7" lang="en"> <![endif]-->
+<!--[if (IE 7)&(!IEMobile)]>  <html class="no-js lt-ie9 lt-ie8" lang="en"> <![endif]-->
+<!--[if IE 8]>                <html class="no-js lt-ie9" lang="en"> <![endif]-->
+<!--[if (gte IE 9)|(gt IEMobile 7)]><!--> <html class="no-js" lang="en"> <!--<![endif]-->
+<head>
```
Comment (RobLoach) [inline generator/handlebars/template/index.html]: Any reason whywe are adding a handlebars generator here? Seems unrelated to the focus mode.

File: builder/handlebars/index.hbs [hunk 2]
```diff
@@ -111,6 +111,9 @@
       {{#if example}}
         <div class="kss-modifier__wrapper">
           <div class="kss-modifier__heading kss-style">
+              <a href="#kssref-{{referenceURI}}-fullscreen"
+                 onclick="var el = document.getElementsByClassName('kss-{{referenceURI}}-fullscreen')[0]; screenfull.toggle(el); return false;"
+                      >Focus</a>
```
Comment (JohnAlbin) [inline builder/handlebars/index.hbs]: I don't see how this will work. If the section has multiple modifiers, getElementsByClassName will return an array of dom elements.

Comment (JohnAlbin) [inline builder/handlebars/index.hbs]: I don't see how this will work. If the section has multiple modifiers, getElementsByClassName will return an array of dom elements.

File: generator/handlebars/template/public/kss.css [hunk 3]
```diff
@@ -0,0 +1 @@
...n{max-width:initial;width:100vw;min-height:100vh;margin-bottom:0;overflow:scroll;padding:0;border:none}#kss-node .pln{color:#000}#kss-node .str{color:#080}#kss-node .kwd{color:#008}#kss-node .com{color:#800}#kss-node .typ{color:#606}#kss-node .lit{color:#066}#kss-node .pun,#kss-node .opn,#kss-node .clo{color:#660}#kss-node .tag{color:#008}#kss-node .atn{color:#606}#kss-node .atv{color:#080}#kss-node .dec,#kss-node .var{color:#606}#kss-node .fun{color:red}@media print,projection{#kss-node .str{color:#060}#kss-node .kwd{color:#006;font-weight:bold}#kss-node .com{color:#600;font-style:italic}#kss-node .typ{color:#404;font-weight:bold}#kss-node .lit{color:#044}#kss-node .pun,#kss-node .opn,#kss-node .clo{color:#440}#kss-node .tag{color:#006;font-weight:bold}#kss-node .atn{color:#404}#kss-node .atv{color:#060}}#kss-node ol.linenums{margin:0;padding:0 0 3px 0;list-style-type:none}#kss-node ol.linenums li{min-height:24px;border-bottom:1px solid #eee;padding:0 10px;background:#fff}#kss-node ol.linenums li:first-child{padding-top:3px}#kss-node ol.linenums li.L0,#kss-node ol.linenums li.L2,#kss-node ol.linenums li.L4,#kss-node ol.linenums li.L6,#kss-node ol.linenums li.L8{background:#fcfcfc}
```
Comment (JohnAlbin) [inline generator/handlebars/template/public/kss.css]: Agree with @RobLoach. Both of these generators/* files look like cruft from rebasing the 2.x work onto the latest 3.x work in master.

File: builder/handlebars/index.hbs (hunk 2 above)
Comment (ryuran) [inline builder/handlebars/index.hbs]: `[0]` will return the first of this collection (maybe not the good one)

Comment (JohnAlbin) [inline builder/handlebars/index.hbs]: ah, yes. I missed the `[0]`, and, yes, showing only the first of multiple examples is still wrong.

**Comment by JohnAlbin** [comment]
I went with a simpler "full window" focus method. https://github.com/kss-node/kss-node/issues/165#issuecomment-258776293
